x-pack/plugins/security_solution/public/common/utils/test_utils.ts

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { ReactWrapper } from 'enzyme';
+import { act } from 'react-dom/test-utils';
+
+// Temporary fix for https://github.com/enzymejs/enzyme/issues/2073
+export const waitForUpdates = async <P>(wrapper: ReactWrapper<P>) => {
+  await act(async () => {
+    await new Promise((resolve) => setTimeout(resolve, 0));
+    wrapper.update();
+  });
+};
```

Comment:
You probably need this extra utility but I usually use a `waitFor()` from here:
https://testing-library.com/docs/dom-testing-library/api-async

When I can before adding a workaround and then I do the workaround when I can't use one of those. Might help you out, might not. Ignore this if it doesn't, just mentioning it in case its helpful.